### DISCUSSION TITLE
Kong 2.8.1 Dockerhub tag

### DISCUSSION THREAD

**Discussion by tilhars** [Kong/kong#8666] [Help] [answered]
Are the changes of 2.8.1 included in the 2.8.0 tag on dockerhub or will there be a Kong 2.8.1 tag available?

**Comment by mayocream** [comment, marked as answer]
Hi @usysrc, there is a `2.8.1` tag on Docker Hub: [`kong:2.8.1`](https://hub.docker.com/layers/kong/library/kong/2.8.1/images/sha256-e7f75a2e891f86caf9a70483ad7cbde119d7594149517fafe619b27bee61d349?context=explore)

**Comment by tilhars** [comment]
Thank you providing it!
